### ISSUE TITLE
Elements of fraction field over multivariate polynomials should be normalized

### ISSUE THREAD

**Issue by maxale** [sagemath/sage#39136]
### Problem Description

Consider an example:
```
sage: R.<x> = QQ[]
sage: 1/(-x)
-1/x
sage: R.<x,y> = QQ[]
sage: 1/(-x)
1/(-x)
```

### Proposed Solution

Multivariate fraction field elements should be normalized similarly to univariate ones. At very least, it should factor out the leading coefficient of denominator when this coefficient is invertible.

### Alternatives Considered

N/A

### Additional Information

See also #38569

### Is there an existing issue for this?

- [X] I have searched the existing issues for a bug report that matches the one I want to file, without success.

**Comment by user202729** [comment]
Probably related to https://github.com/sagemath/sage/pull/38924 .

tl;dr: Easier said than done.

**Comment by maxale** [comment]
> And there are other cases like `(3*x+1)/(4*x+1)` in `Frac(QQ[x])` you don't want to normalize like that.

Such cases are related to https://github.com/sagemath/sage/issues/37544#issuecomment-1986419749

**Comment by maxale** [comment]
Possible related #38569
